src/platform/packages/shared/response-ops/alerts-fields-browser/components/field_items/field_items.tsx

```diff
@@ -67,28 +67,30 @@ export const getFieldItemsData = ({
     selectedCategoryIds.length > 0 ? selectedCategoryIds : Object.keys(browserFields);
   const selectedFieldIds = new Set(columnIds);
 
-  const fieldItems = uniqBy(
-    'name',
-    categoryIds.reduce<BrowserFieldItem[]>((fieldItemsAcc, categoryId) => {
+  const getFieldItems = () => {
+    let fieldItemsAcc: BrowserFieldItem[] = [];
+    for (let i = 0; i < categoryIds.length; i += 1) {
+      const categoryId = categoryIds[i];
       const categoryBrowserFields = Object.values(browserFields[categoryId]?.fields ?? {});
       if (categoryBrowserFields.length > 0) {
-        fieldItemsAcc.push(
-          ...categoryBrowserFields.map(({ name = '', ...field }) => {
-            return {
-              name,
-              type: field.type,
-              description: getDescription(name, EcsFlat as Record<string, EcsMetadata>),
-              example: field.example?.toString(),
-              category: getCategory(name),
-              selected: selectedFieldIds.has(name),
-              isRuntime: !!field.runtimeField,
-            };
-          })
-        );
+        const categoryFieldItems = categoryBrowserFields.map(({ name = '', ...field }) => {
+          return {
+            name,
+            type: field.type,
+            description: getDescription(name, EcsFlat as Record<string, EcsMetadata>),
+            example: field.example?.toString(),
+            category: getCategory(name),
+            selected: selectedFieldIds.has(name),
+            isRuntime: !!field.runtimeField,
+          };
+        });
+        fieldItemsAcc = fieldItemsAcc.concat(categoryFieldItems);
```

Comment:
```suggestion
       let fieldSeen = {} // this can go on line 71
        for (let j = 0; j < categoryBrowserFields.length; j += 1) {
          const field = categoryBrowserFields[j];
          const name = field.name || '';
          if(name in fieldSeen) continue;
          fieldSeen[name] = true
          const categoryFieldItem = {
            name,
            type: field.type,
            description: getDescription(name, EcsFlat as Record<string, EcsMetadata>),
            example: field.example?.toString(),
            category: getCategory(name),
            selected: selectedFieldIds.has(name),
            isRuntime: !!field.runtimeField,
          };
          fieldItemsAcc.push(categoryFieldItem);
        }

```


What do you think about above suggestion. With this we can get rid of `uniqBy` and `concat` which needs another iteration.

Reply:
I like it. 👏🏾 Will make the change

Reply:
Done, with some very minor tweaks!